lib/linguist/languages.yml

```diff
@@ -435,6 +435,17 @@ Awk:
   tm_scope: source.awk
   ace_mode: text
   language_id: 28
+B4X:
+  type: programming
+  color: "#5a6986"
+  extensions:
+  - ".b4a"
+  - ".b4j"
+  - ".b4i"
+  - ".b4r"
```

Comment:
Please remove the extensions that you've stated don't meet the usage requirements